Tests/Runtime/TimingSystem/Tests/AbstractTimerTests.cs

```diff
@@ -0,0 +1,93 @@
+
+```csharp
```

Comment:
Abstract timer is a monobehaviour so you have to create an object and add it as a component to use it. You also need to destroy object when done. Please fix